### PR TITLE
WW-4582 Permanent patch for security issue CVE-2014-0094 adds 'class' to exclude

### PR DESCRIPTION
**Pull request by victorsosa**
adds 'class' to exclude params in ParametersInterceptor (avoid ClassLoader manipulation)

### CODE REVIEW

File: core/src/test/java/com/opensymphony/xwork2/security/DefaultExcludedPatternsCheckerTest.java [hunk 1]
```diff
@@ -11,6 +11,9 @@
     public void testHardcodedPatterns() throws Exception {
         // given
         List<String> params = new ArrayList<String>() {
+
+            private static final long serialVersionUID = 5687184571054993717L;
```
Comment (lukaszlenart) [inline core/src/test/java/com/opensymphony/xwork2/security/DefaultExcludedPatternsCheckerTest.java]: Could you disable auto-generation of `serialVersionUID`?

Comment (victorsosa) [inline core/src/test/java/com/opensymphony/xwork2/security/DefaultExcludedPatternsCheckerTest.java]: OK, I will remove that line

**Comment by victorsosa** [comment]
This close also the CVE-2014-0112, CVE-2014-0113 and CVE-2014-0116

**Comment by lukaszlenart** [comment]
This isn't really needed as access to `Class` is blocked for any Ognl expression, see [Internal Security Mechanism](http://struts.apache.org/docs/security.html#Security-Internalsecuritymechanism)

**Comment by lukaszlenart** [comment]
Also based on a report from a user I have added this 74e26830d2849a84729b33497f729e0f033dc147 - this is almost the same as yours but it bases on a real-life example :)

**Comment by victorsosa** [comment]
ok, PR closed

**Pull request closed without merging**